frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tag, Table } from '@trussworks/react-uswds';
+import { Link } from 'react-router-dom';
+import SimpleBar from 'simplebar-react';
+import 'simplebar/dist/simplebar.min.css';
+
+import Container from '../../components/Container';
+import NewReport from './NewReport';
+import 'uswds/dist/css/uswds.css';
+import '@trussworks/react-uswds/lib/index.css';
+import './index.css';
+
+function renderReports(reports) {
+  return reports.map((report) => {
+    const {
+      id,
+      displayId,
+      activityRecipients,
+      startDate,
+      author,
+      collaborators,
+      status,
+    } = report;
+
+    const recipientsTitle = activityRecipients.reduce(
+      (result, ar) => `${result + (ar.grant ? ar.grant.grantee.name : ar.name)}\n`,
+      '',
+    );
+
+    const recipients = activityRecipients.map((ar) => (
+      <Tag
+        key={ar.name.slice(1, 3) + ar.id}
+        className="smart-hub--table-collection"
+      >
+        {ar.grant ? ar.grant.grantee.name : ar.name}
+      </Tag>
+    ));
+
+    const collaboratorsTitle = collaborators.reduce(
+      (result, collaborator) => `${result + collaborator.fullName}\n`,
+      '',
+    );
+
+    const collaboratorsWithTags = collaborators.map((collaborator) => (
+      <Tag
+        key={collaborator.fullName.slice(1, 13)}
+        className="smart-hub--table-collection"
+      >
+        {collaborator.fullName}
+      </Tag>
+    ));
+
+    return (
+      <tr key={`my_alerts_${id}`}>
+        <td>
+          <Link
+            to={`/activity-reports/${id}/activity-summary`}
```

Comment:
If the `/activity-summary` is removed the user will be redirected to the `review` page if the report is not in draft mode (the report is waiting for approval or "Needs Action")